packages/opc-comment-input/src/opc-comment-input.ts

```diff
@@ -0,0 +1,101 @@
+import { LitElement, html, property, customElement, query, internalProperty } from 'lit-element';
+import style  from './opc-comment-input.scss';
+
+@customElement('opc-comment-input')
+export class CommentInput extends LitElement {
+  @property({type: String})
+  placeholder = 'Type your comment';
+
+  @internalProperty()
+  protected commentText = '';
+
+  @query('textarea')
+  textarea: HTMLTextAreaElement;
+
+  static get styles() {
+    return [ style ];
+  }
+
+  _resetCommentInput() {
+    this.commentText = '';
+    this.textarea.value = '';
+  }
+
+  render() {
+    return html`
+      <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly/patternfly.css" crossorigin="anonymous">
+      <div class="pf-c-form">
+        <div class="pf-c-form__group">
+          <div class="pf-c-form__group-label">
+            <label class="pf-c-form__label" for="vertical-align-labels-form-name">
+              <span class="pf-c-form__label-text">Add Comment</span>
+            </label>
+          </div>
+          <div class="pf-c-form__group-control">
+            <textarea class="pf-c-form-control"
+              type="text"
+              aria-label="textarea example"
+              placeholder="${ this.placeholder }"
+              @input="${(e: HTMLElementEventMap | any) => this.commentText = e.target.value}"></textarea>
+          </div>
+        </div>
+        <div class="pf-c-form__actions pf-l-flex pf-m-justify-content-flex-end">
+          <button class="pf-c-button pf-m-secondary" type="reset"
+            @click="${this._resetCommentInput}"
+            style="--pf-c-button--m-secondary--Color: #000; --pf-c-button--after--BorderColor: transparent;">
+            Cancel
+          </button>
+          <button class="pf-c-button pf-m-primary"
+            ?disabled="${!this.commentText.length}"
+            @click="${(e: HTMLElementEventMap) => this._commentButtonClicked()}">Comment</button>
+        </div>
+      </div>
+    `;
+  }
+
+  _commentButtonClicked() {
+    if (this.commentText.length) {
+      let commentSubmitEvent = new CustomEvent('opc-comment-button:click', {
+        detail: {
+          message: 'opc-comment-submit button is clicked',
+          data: {
+            commentText: this.commentText,
+          }
+        }
+      });
+      this.dispatchEvent(commentSubmitEvent);
+      this._resetCommentInput();
+    }
+  }
+}
+
+@customElement('opc-comment-list')
+export class CommentList extends LitElement {
+
+  @property({type: Array})
+  comments: OPCComment[] = [];
+
+  static get styles() {
+    return [ style ];
+  }
+
+  render() {
+    return html`
+      ${ !this.comments.length ?
+        html`
+          <div class="comment--empty__state">
+            <p><strong>No comments found...</strong> but you can be the first to start the conversations.</p>
```

Comment:
rephrase this sentence to `No comments yet! Type your comment to start the conversation`